tests/test-dirs/type-enclosing/cons.t

```diff
@@ -0,0 +1,62 @@
+Various parts of the cons.ml:
+
+- The expression:
+  $ $MERLIN single type-enclosing -position 4:14 -verbosity 0 \
+  > -filename ./cons.ml < ./cons.ml | jq ".value[0:2]"
+  [
+    {
+      "start": {
+        "line": 4,
+        "col": 13
+      },
+      "end": {
+        "line": 4,
+        "col": 14
+      },
+      "type": "t",
+      "tail": "no"
+    },
+    {
+      "start": {
+        "line": 4,
+        "col": 13
+      },
+      "end": {
+        "line": 4,
+        "col": 14
+      },
+      "type": "t",
+      "tail": "no"
+    }
+  ]
```

Comment:
Perhaps you should add a comment explaining why the output is duplicated, and that we aim to fix that in the future.